### PR TITLE
Change build reqs from VS update 1 to 2

### PR DESCRIPTION
**Pull request by agocke**
/cc @dotnet/roslyn Please review

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
Heh, sounds good. 👍

File: docs/contributing/Building, Debugging, and Testing on Windows.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-1. [Visual Studio 2015 with Update 1](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 1_.
+1. [Visual Studio 2015 with Update 2](http://go.microsoft.com/fwlink/?LinkId=691129). _You need Update 2_.
 2. Visual Studio 2015 Extensibility Tools.
  
     If you already installed Visual Studio, the Extensibility Tools can be added as follows: 
     - Open Control Panel -> Programs and Features
     - Select the entry for your installation of Microsoft Visual Studio. Depending on your version, it may appear as follows:
-        - Microsoft Visual Studio Community 2015 with Update 1
+        - Microsoft Visual Studio Community 2015 with Update 2
```
Comment (leppie) [inline docs/contributing/Building, Debugging, and Testing on Windows.md]: Maybe generalize this?

**Comment by kayle** [comment]
@agocke Is update 2 *really* required, or will update 3 work now as well? If update 2, is there another issue for supporting update 3?

**Comment by agocke** [comment]
@kayle Update 2+ should work. This commit was made before Update 3 existed :)

**Pull request merged**